src/Entity/Arrival.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Ups\Entity;
+
+use DOMDocument;
+use DOMElement;
+use Ups\NodeInterface;
+
+/**
+ * @author mazzarito
+ */
+class Arrival
+{
+    /**
+     * @var string
+     */
+    private $date;
+
+    /**
+     * @var string
+     */
+    private $time;
+
+    /**
+     * @param \stdClass|null $response
+     */
+    public function __construct(\stdClass $response = null)
+    {
+        if (null !== $response) {
+            if (isset($response->Date)) {
+                $this->date = $response->Date;
+            }
+            if (isset($response->Time)) {
+                $this->time = $response->Time;
+            }
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function getDate()
+    {
+        return $this->date;
+    }
+
+    /**
+     * @param string $date
+     */
+    public function setDate($date)
+    {
+        $this->date = $date;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTime()
+    {
+        return $this->time;
+    }
+
+    /**
+     * @param string $time
+     */
+    public function setTime($time)
```

Comment:
Same goes here. Or we could even move them together in a single method/property if they always come together.